test/Generator/PropertyGeneratorTest.php

```diff
@@ -296,8 +300,10 @@ public function testSetDefaultValue(string $type, $value): void
         $property = new PropertyGenerator();
         $property->setDefaultValue($value, $type);
 
-        self::assertEquals($type, $property->getDefaultValue()->getType());
-        self::assertEquals($value, $property->getDefaultValue()->getValue());
+        $valueGenerator = $property->getDefaultValue();
+        self::assertNotNull($valueGenerator);
+        self::assertEquals($type, $valueGenerator->getType());
+        self::assertEquals($value, $valueGenerator->getValue());
```

Comment:
Should be `assertSame` to avoid type juggling issues

Reply:
Done